_docs/_latest/setup/install.md

```diff
@@ -42,31 +42,37 @@ steps below:
 
 ### Run setup
 
-  1. After you've started Cloud Shell, download Forseti. The installer is
-     included. Getting `2.0-dev-rc1` branch will install Forseti v2.0 Release
-     Candidate 1.
+  1. After you've started Cloud Shell, download Forseti. The installer is included. 
```

Comment:
Please put a `---` before the first section.

Reply:
Updated